vending-app/src/main/aidl/com/android/vending/billing/IInAppBillingIsAlternativeBillingOnlyAvailableCallback.aidl

```diff
@@ -0,0 +1,18 @@
+// IInAppBillingIsAlternativeBillingOnlyAvailableCallback.aidl
+/*
+ * SPDX-FileCopyrightText: 2023 microG Project Team
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.android.vending.billing;
+
+import android.os.Bundle;
+
+// Declare any non-default types here with import statements
+
+interface IInAppBillingIsAlternativeBillingOnlyAvailableCallback {
+    /**
```

Comment:
We don't need these instruction comments in our AIDL files.
Also the first line with the file name is very unnecessary.